### ISSUE TITLE
Edit field layout of block without having to group blocks?

### ISSUE THREAD

**Issue by benjamminf** [supercool/Pimp-My-Matrix#28]
Currently in order to use the tabbing feature on individual blocks, they have to be assigned to a group. This becomes a minor hassle when only using a handful of block types, where having a drop down menu to add blocks is unnecessary.

Maybe an idea would be to not show a drop down menu if a block group only has one block in it?

**Comment by joshangell** [comment]
Hey Benjamin - that was something we considered and just went with the current behaviour as it suited our needs. I'd be happy to add a setting to change the behaviour if there is only one group if you like?

I did start thinking about how I could have field layouts without requiring a group, but things got over complex fast so I gave up on that ...

**Comment by benjamminf** [comment]
I definitely see why this is a complex problem to solve. And I understand my suggestion is a little too hacky for such a high quality plugin ;) The only other solution I could think of is a simple lightswitch under the "Group your block types" editor, asking if groups should be shown. Again, not ideal so I understand if this is a feature not worth implementing.

**Comment by joshangell** [comment]
Yeah - the thing that frustrates me is it feels pretty hacky to have to have a group in the first place ... so I've kind of already lost there! The lightswitch in context sounds better to me than a global preference, and allows for a more granular setup.

I'll leave this issue open as a FR and hopefully I'll get some time to take a proper look. Thanks for your input :smile:
